examples/A11ySample/src/screens/Basic/DynamicOrderScreen/DynamicOrderScreen.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useCallback, useState } from "react";
+import { useFocusEffect, useNavigation } from "@react-navigation/native";
+import { StyleSheet, Text } from "react-native";
+import { A11yOrder, useDynamicFocusOrder } from "react-native-a11y";
+
+import {
+  STATUS_SCREEN,
+  DrawerNavigation,
+  REF_MANAGEMENT,
+} from "../../../../navigation";
+import { Button, NavBar, ReaderExample, Screen } from "../../../components";
+
+export const DynamicOrderScreen = () => {
+  const navigation = useNavigation<DrawerNavigation>();
+  const goBack = () => navigation.navigate(STATUS_SCREEN);
+  const goNext = () => navigation.navigate(REF_MANAGEMENT);
+  const [showThird, setState] = useState(true);
+
+  const { a11yOrder, registerOrder, setOrder } = useDynamicFocusOrder();
+  useFocusEffect(
```

Comment:
maybe this is redundant? 28 line is the same. Also, maybe it can be moved to a util hook?

Reply:
My bad.

Reply:
done